src/main/java/org/fiximulator/core/FIXimulatorApplication.java

```diff
@@ -572,6 +572,15 @@ public void sendExecution(Execution execution) {
             executionReport.set(new SecurityID(order.getSecurityID()));
             executionReport.set(new IDSource(order.getIdSource()));
         }
+        System.out.println("LiqudityFlag");
+        boolean sendLiquidityFlag = false;
+        try {
+            sendLiquidityFlag = settings.getBool("FIXimulatorSendLiquidityFlag");
+        } catch (Exception ex) {}
```

Comment:
Prefer to avoid catching bare `Exception` whenever possible

Reply:
⬆️ mystery catch-all leads to trouble

Reply:
👋 @kaiserleib